src/lib/systemlib/system_params.c

```diff
@@ -100,8 +100,8 @@ PARAM_DEFINE_INT32(SYS_RESTART_TYPE, 2);
  *
  * Set the group of estimators used for multicopters and VTOLs
  *
- * @value 1 local_position_estimator, attitude_estimator_q
- * @value 2 ekf2
+ * @value 1 local_position_estimator, attitude_estimator_q (unsupported)
```

Comment:
Can we add a pure q estimator option for minimal setups?

Reply:
I think as long as we could guarantee it wouldn't bit rot, that would be a good feature. Also for research/development I think stabilizing the interface for estimation (ekf2 module) and control (attitude_control /position_control module) into an API with swap-able code-generated back-ends would be nice. I have something along those lines that I hope to finish this summer at least for the estimation side.

Reply:
@bkueng yes, let's followup in another PR.